packages/cascara/src/ui/Chat/ChatAttachment.js

```diff
@@ -79,7 +80,7 @@ const ChatAttachment = ({
             {type}
           </div>
           <FilesEmptyIcon outline />
-        </div>
+        </Role>
       }
       onClick={handleDownloadAttachment}
     />
```

Comment:
Could this be something like the following?

```javascript
<Role
      as={Attachment} // please note we are using the as prop of Role
      actionable
      description={bytesToSize(size)}
      header={fileName}
      icon={
        // TODO: Extract these to CSS modules
        <Role
          style={{
            position: 'relative',
          }}
        >
          <div
            style={{
              bottom: '.375em',
              fontSize: '.75em',
              position: 'absolute',
              textAlign: 'center',
              width: '100%',
              zIndex: 1,
            }}
          >
            {type}
          </div>
          <FilesEmptyIcon outline />
        </Role>
      }
      onClick={handleDownloadAttachment}
    />
```

Reply:
The above comment isn't a change request either. The code is fine 👍 

@rociobaena, @brianespinosa what do you think about the pattern described above?

Reply:
Manu, at the beginning I thought that wouldn't work, but it is working. I think I like it.

Reply:
awesome, if it works and everybody agrees, let's keep it 🙌